localstripe/resources.py

```diff
@@ -494,6 +494,29 @@ def _api_delete(cls, id):
         schedule_webhook(Event('customer.deleted', obj))
         return super()._api_delete(id)
 
+    @classmethod
+    def _api_retrieve_source(cls, id, source_id, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        source_obj = None
+
+        # return 404 if does not exist
+        if type(source_id) is str and source_id.startswith('src_'):
+            source_obj = Source._api_retrieve(source_id)
+        elif type(source_id) is str and source_id.startswith('tok_'):
+            source_obj = Token._api_retrieve(source_id).card
```

Comment:
Tokens can be associated to customers sources? It seems strange. Usually they're just used to create a new source.